refactor creating (#176)
Effect-TS/website: pages/docs/essentials/creating.mdx

# Creating Effects

Effect provides different ways to create effects, which are units of computation that encapsulate side effects.
In this tutorial, we will cover some of the common methods that you can use to create effects.

> ## Why Not Throwing Errors?

Reviewer: ```suggestion
## Why Not Throw Errors?
```

- fails with an expected error of type `string`
- does not produce any value (`never`)

> By using `Effect.succeed` and `Effect.fail`, you can explicitly handle success and failure cases and the type system ensures that errors are tracked and accounted for.

Reviewer: ```suggestion
With `Effect.succeed` and `Effect.fail`, you can explicitly handle success and failure cases and the type system will ensure that errors are tracked and accounted for.
```
